src/App.css

```diff
@@ -1,13 +1,20 @@
 .App {
   text-align: center;
-  background-color: darkgray;
+  background-color: #666666;
 }
 
 .App-logo {
   animation: App-logo-spin infinite 20s linear;
   height: 80px;
 }
 
+.columnOfBoxes {
```

Comment:
You add styled components. You do not need this, please use styled-components for css declarations,